flow/flow.go

```diff
@@ -905,6 +905,18 @@ func (f *Flow) newApplicationLayer(packet *Packet, opts Opts) error {
 				NSCount: d.NSCount,
 				QDCount: d.QDCount,
 			}
+			questions := make([]string, len(d.Questions))
```

Comment:
as per the current implementation, this need to go inside the if block like answers.

Reply:
why? Isn't there always a question in a DNS packet? even the response includes the question as far as I know so the "if" statement would be redundant..? isn't this correct?

Reply:
if some question present always no need for the ```if```

Reply:
in some special case the question may be ```zero``` as well. http://www-inf.int-evry.fr/~hennequi/CoursDNS/NOTES-COURS_eng/msg.html i refered this site, correct me if i am wrong.

Reply:
You're probably right for there could be very special cases where no questions exist. changed this